graphs/frequent_pattern_graph_miner.py

```diff
@@ -0,0 +1,192 @@
+'''
+FP-GraphMiner - A Fast Frequent Pattern Mining Algorithm for Network Graphs
+
+A novel Frequent Pattern Graph Mining algorithm,
+FP-GraphMiner, that compactly represents a set of network graphs as a Frequent Pattern Graph (or FP-Graph).
+This graph can be used to efficiently mine frequent subgraphs including maximal frequent subgraphs and maximum common subgraphs.
+
+URL:https://www.researchgate.net/publication/235255851_FP-GraphMiner_-_A_Fast_Frequent_Pattern_Mining_Algorithm_for_Network_Graphs
+
+'''
+from typing import List
+
+def get_distinct_edge(edge_array: List[List[str]]) -> List[str]:
+    '''
+    Return Distinct edges from edge array of multiple graphs
+    '''

```

Comment:
Let's add a doctest _inside_ the docstring of the function.  The first test should pass and the second should fail.
```suggestion
    >>> sorted(get_distinct_edge(edge_array))
    ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h']
    >>> sorted(get_distinct_edge(edge_array))
    ['z', 'b', 'c', 'd', 'e', 'f', 'g', 'h']
    '''

```

Reply:
Now if we do `python3 -m doctest -v graphs/frequent_pattern_graph_miner.py` we should see the test results.

Reply:
The three >>> and the space after them are significant.

Reply:
Can we please do this one so that we have at least one doctest running?